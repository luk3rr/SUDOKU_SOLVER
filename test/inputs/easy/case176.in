904000070 800200000 120400000 040708060 030100004 009040000 060000500 001060000 000037090 
