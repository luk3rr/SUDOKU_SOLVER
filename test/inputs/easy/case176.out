Solving the following grid:
9 0 4 | 0 0 0 | 0 7 0 
8 0 0 | 2 0 0 | 0 0 0 
1 2 0 | 4 0 0 | 0 0 0 
------+-------+------
0 4 0 | 7 0 8 | 0 6 0 
0 3 0 | 1 0 0 | 0 0 4 
0 0 9 | 0 4 0 | 0 0 0 
------+-------+------
0 6 0 | 0 0 0 | 5 0 0 
0 0 1 | 0 6 0 | 0 0 0 
0 0 0 | 0 3 7 | 0 9 0 


Solution found :')

9 5 4 | 6 8 1 | 2 7 3 
8 7 6 | 2 5 3 | 4 1 9 
1 2 3 | 4 7 9 | 6 8 5 
------+-------+------
2 4 5 | 7 9 8 | 3 6 1 
7 3 8 | 1 2 6 | 9 5 4 
6 1 9 | 3 4 5 | 8 2 7 
------+-------+------
3 6 7 | 9 1 2 | 5 4 8 
5 9 1 | 8 6 4 | 7 3 2 
4 8 2 | 5 3 7 | 1 9 6 

Algorithm: BFS
Total time: 131 ms
Total expanded states: 21901
